src/app/teacher/edit-tag/edit-tag.component.ts

```diff
@@ -56,6 +63,12 @@ export class EditTagComponent {
     });
   }
 
+  ngAfterViewInit(): void {
+    setTimeout(() => {
+      (<any>this.nameInput).nativeElement.focus();
```

Comment:
Is there a need to cast?
```suggestion
      this.nameInput.nativeElement.focus();
```